CMakeLists.txt

```diff
@@ -52,6 +52,8 @@ if (WITH_GFLAGS)
 
   if (gflags_FOUND)
     set (HAVE_LIB_GFLAGS 1)
+    # When linking unittest, mingw32 cannot find -lgflags without this.
+    LINK_DIRECTORIES( ${LINK_DIRECTORIES} ${gflags_INCLUDE_DIR}/../lib)
```

Comment:
Besides being inconsistent with the CMake style used, your gflags installation is either broken, or the gflags CMake config needs to be patched for MinGW. `target_link_libraries` already takes cares of including the correct link directories. So this change shouldn't be needed.